react-native/react/native/pinentry-listeners.desktop.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  'keybase.1.secretUi.getPassphrase': (payload, response) => {
+    console.log('Asked for passphrase')
+    /*
```

Comment:
@MarcoPolo I tried :(  This file imports BrowserWindow and ipc, and they live outside the react-native directory and flow gets sad.